src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -156,6 +157,9 @@ def collect_data(geop_result, geojson):
 
     z.AvSlope = geop_result['avg_slope']
 
+    z.LS = (ls_factors(geop_result['lu_stream_pct'], z.StreamLength,
```

Comment:
Removing them starts a guessing-game with flake of how many indents is correct for a continuation.